optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -843,8 +843,8 @@
 
       <para>Requires the score calculation to support <link linkend="explainingTheScore">constraint matches</link>.
       <link linkend="droolsScoreCalculation">Drools score calculation</link> supports constraint matches automatically,
-      but <link linkend="incrementalJavaScoreCalculation">incremental Java score calculation</link> requires requires
-      more work.</para>
+      but <link linkend="incrementalJavaScoreCalculation">incremental Java score calculation</link> requires more work.
+      </para>
```

Comment:
Done, by opening and saving in a docbook editor with our coding style. Thanks, Ondrej.
  http://github.com/droolsjbpm/optaplanner/commit/ee521979b

Note to Jiri: this is a nitpick, which means it will never stop a contributor's PR from being merged, so don't let it discourage you to make future contributions :) Thanks for this PR!